include/deal.II/grid/grid_tools_cache.h

```diff
@@ -173,6 +204,20 @@ namespace GridTools
     mutable KDTree<spacedim> vertex_kdtree;
 #endif
 
+#ifdef DEAL_II_WITH_MPI
+    /**
+     * Store the used vertices of the Triangulation, as generated by
+     * GridTools::extract_used_vertices().
+     */
+    mutable std::vector< std::vector< BoundingBox<spacedim> > > global_bounding_boxes;
```

Comment:
And make these variables always available -- they would simply not be used in the non-MPI case.

Reply:
Separately, every `mutable` variable needs to be guarded by a mutex, but I don't see it here (or see it in the documentation of the variable).

Reply:
Finally, the doc string does not document this variable.